drivers/display/display_ili9340.c

```diff
@@ -62,46 +65,46 @@ static int ili9340_init(struct device *dev)
 	data->spi_config.slave = DT_INST_0_ILITEK_ILI9340_BASE_ADDRESS;
 
 #ifdef DT_INST_0_ILITEK_ILI9340_CS_GPIOS_CONTROLLER
-	data->cs_ctrl.gpio_dev =
+	data->cs.gpio_dev =
 		device_get_binding(DT_INST_0_ILITEK_ILI9340_CS_GPIOS_CONTROLLER);
-	data->cs_ctrl.gpio_pin = DT_INST_0_ILITEK_ILI9340_CS_GPIOS_PIN;
-	data->cs_ctrl.delay = 0U;
-	data->spi_config.cs = &(data->cs_ctrl);
+	data->cs.gpio_pin = DT_INST_0_ILITEK_ILI9340_CS_GPIOS_PIN;
+	data->cs.delay = 0U;
+	data->spi_config.cs = &(data->cs);
 #else
 	data->spi_config.cs = NULL;
 #endif
 
 #ifdef DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_CONTROLLER
-	data->reset_gpio =
+	data->rst =
 		device_get_binding(DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_CONTROLLER);
-	if (data->reset_gpio == NULL) {
+	if (data->rst == NULL) {
 		LOG_ERR("Could not get GPIO port for ILI9340 reset");
 		return -EPERM;
 	}
 
-	gpio_pin_configure(data->reset_gpio, DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_PIN,
+	gpio_pin_configure(data->rst, DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_PIN,
 			   GPIO_OUTPUT_INACTIVE |
 			   DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_FLAGS);
 #endif
 
-	data->command_data_gpio =
+	data->cmd =
 		device_get_binding(DT_INST_0_ILITEK_ILI9340_CMD_DATA_GPIOS_CONTROLLER);
-	if (data->command_data_gpio == NULL) {
+	if (data->cmd == NULL) {
 		LOG_ERR("Could not get GPIO port for ILI9340 command/data");
 		return -EPERM;
 	}
 
-	gpio_pin_configure(data->command_data_gpio, DT_INST_0_ILITEK_ILI9340_CMD_DATA_GPIOS_PIN,
-			   GPIO_OUTPUT |
+	gpio_pin_configure(data->cmd, DT_INST_0_ILITEK_ILI9340_CMD_DATA_GPIOS_PIN,
+			   GPIO_OUTPUT_ACTIVE | GPIO_ACTIVE_LOW |
```

Comment:
Not necessary to change it, the flag like GPIO_ACTIVE_LOW would be obtained from *_GPIOS_FLAGS

Reply:
Make sense. My idea was to "force' inversion, but with this change nobody will able to change. thnx, will remove